other/genetic_algorithm.py

```diff
@@ -0,0 +1,79 @@
+import random
+
+population = 200
+poolLength = 20
+generations = 0
+mutation = 0.01
+
+alphabet = "abcdefghijklmnopqrstuvwxyz! "
+target = "target text"
+output = ""
+data = []
+pool = []
+score_range = []
+
+class Item:
+    def __init__(self, data, target):
+        self.target = target
+        self.data = data
+        self.score = self.get_score()
+
+    def get_score(self):
+        score = 0
+        for i in range(len(self.data)):
+            if self.data[i] == self.target[i]:
+                score += 1
+        return score // len(self.data)
```

Comment:
Change `int division` to `float division` as the code gets stuck in infinite loop. Therefore no improvement is made in new generations.
`return score / len(self.data)`